diff --git a/mojo/services/view_manager/BUILD.gn b/mojo/services/view_manager/BUILD.gn
index 4ea34d5..e1f66ae 100644
--- a/mojo/services/view_manager/BUILD.gn
+++ b/mojo/services/view_manager/BUILD.gn
@@ -44,6 +44,10 @@ source_set("view_manager_lib") {
     "window_manager_access_policy.h",
   ]
 
+  public_deps = [
+    "//mojo/services/public/cpp/view_manager:common",
+  ]
+
   deps = [
     "//base",
     "//cc/surfaces",
