diff --git a/mojo/public/tools/bindings/mojom.gni b/mojo/public/tools/bindings/mojom.gni
index e421cf2..80ff5ae 100644
--- a/mojo/public/tools/bindings/mojom.gni
+++ b/mojo/public/tools/bindings/mojom.gni
@@ -23,9 +23,6 @@ import("//build/config/mojo.gni")
 #
 #   public_configs (optional)
 #
-#   mojom_import_dirs (optional)
-#       List of directories that will be searched for imported mojom files.
-#
 #   testonly (optional)
 #
 #   visibility (optional)
@@ -134,6 +131,8 @@ template("mojom") {
       "-o", rebase_path(root_gen_dir),
     ]
 
+    # The user might have specified additional mojom import dirs in
+    # //build/config/mojo.gni.
     if (defined(mojom_import_dirs)) {
       foreach(d, mojom_import_dirs) {
         args += ["-I", rebase_path(d, root_build_dir)]
