diff --git a/mojo/services/public/DEPS b/mojo/services/public/DEPS
new file mode 100644
index 0000000..ce461e4
--- /dev/null
+++ b/mojo/services/public/DEPS
@@ -0,0 +1,4 @@
+include_rules = [
+  # TODO(blundell): What is the right solution here?
+  "+services/public",
+]
diff --git a/mojo/services/public/cpp/view_manager/tests/view_manager_unittest.cc b/mojo/services/public/cpp/view_manager/tests/view_manager_unittest.cc
index 3e416fa..5e1db6e 100644
--- a/mojo/services/public/cpp/view_manager/tests/view_manager_unittest.cc
+++ b/mojo/services/public/cpp/view_manager/tests/view_manager_unittest.cc
@@ -13,6 +13,7 @@
 #include "mojo/public/cpp/application/application_impl.h"
 #include "mojo/public/cpp/application/service_provider_impl.h"
 #include "mojo/public/interfaces/application/service_provider.mojom.h"
+#include "mojo/shell/shell_test_helper.h"
 #include "services/public/cpp/geometry/geometry_util.h"
 #include "services/public/cpp/view_manager/lib/view_manager_client_impl.h"
 #include "services/public/cpp/view_manager/lib/view_private.h"
@@ -20,7 +21,6 @@
 #include "services/public/cpp/view_manager/view_manager_client_factory.h"
 #include "services/public/cpp/view_manager/view_manager_delegate.h"
 #include "services/public/cpp/view_manager/view_observer.h"
-#include "mojo/shell/shell_test_helper.h"
 #include "testing/gtest/include/gtest/gtest.h"
 
 namespace mojo {
