diff --git a/build/get_landmines.py b/build/get_landmines.py
index 1eec5ee..9491879 100755
--- a/build/get_landmines.py
+++ b/build/get_landmines.py
@@ -53,6 +53,7 @@ def print_landmines():
   if (platform() != 'ios'):
     print 'Clobber to get rid of obselete test plugin after r248358'
     print 'Clobber to rebuild GN files for V8'
+  print 'Clobber to get rid of stale generated mojom.h files'
   print 'Need to clobber everything due to build_nexe change in nacl r13424'
   print '[chromium-dev] PSA: clobber build needed for IDR_INSPECTOR_* compil...'
   print 'blink_resources.grd changed: crbug.com/400860'
