diff --git a/third_party/mojo/src/mojo/edk/js/test/BUILD.gn b/third_party/mojo/src/mojo/edk/js/test/BUILD.gn
index 6e9792c..badc3ad 100644
--- a/third_party/mojo/src/mojo/edk/js/test/BUILD.gn
+++ b/third_party/mojo/src/mojo/edk/js/test/BUILD.gn
@@ -8,13 +8,13 @@ test("js_unittests") {
     "../../js:js_unittests",
     "../../test:run_all_unittests",
     "../../test:test_support",
-    "../../../environment:chromium",
     "../../../public/cpp/environment",
     "../../../public/cpp/system",
     "../../../public/cpp/utility",
     "../../../public/interfaces/bindings/tests:test_interfaces",
     "//base",
     "//gin:gin_test",
+    "//mojo/environment:chromium",
   ]
 
   sources = [
@@ -28,11 +28,11 @@ test("js_integration_tests") {
     "../../js/tests:js_to_cpp_tests",
     "../../test:run_all_unittests",
     "../../test:test_support",
-    "../../../environment:chromium",
     "../../../public/cpp/bindings",
     "../../../public/interfaces/bindings/tests:test_interfaces",
     "//base",
     "//gin:gin_test",
+    "//mojo/environment:chromium",
   ]
 
   sources = [
