diff --git a/mojo/environment/BUILD.gn b/mojo/environment/BUILD.gn
index e10850a..7843d5f 100644
--- a/mojo/environment/BUILD.gn
+++ b/mojo/environment/BUILD.gn
@@ -10,8 +10,8 @@ source_set("chromium") {
     "environment.cc",
 
     # TODO(vtl): This is kind of ugly. (See TODO in logging.h.)
-    "../public/cpp/environment/logging.h",
-    "../public/cpp/environment/lib/logging.cc",
+    "//third_party/mojo/public/cpp/environment/logging.h",
+    "//third_party/mojo/public/cpp/environment/lib/logging.cc",
   ]
 
   public_deps = [
@@ -35,6 +35,8 @@ component("chromium_impl") {
 
   defines = [ "MOJO_ENVIRONMENT_IMPL_IMPLEMENTATION" ]
 
+  public_configs = [ "//third_party/mojo/public/build/config:mojo_sdk" ]
+
   deps = [
     "//base",
     "//base/third_party/dynamic_annotations",
