diff --git a/chrome/common/extensions/api/BUILD.gn b/chrome/common/extensions/api/BUILD.gn
index a760cd2..2b538f9 100644
--- a/chrome/common/extensions/api/BUILD.gn
+++ b/chrome/common/extensions/api/BUILD.gn
@@ -21,6 +21,10 @@ json_schema_api("api_registration") {
   bundle_registration = true
 
   deps = [
+    # Different APIs include headers from these targets.
+    "//content/public/browser",
+    "//extensions/browser",
+
     # Different APIs include some headers from chrome/common that in turn
     # include generated headers from these targets.
     # TODO(brettw) this should be made unnecessary if possible.
diff --git a/device/serial/BUILD.gn b/device/serial/BUILD.gn
index 0161cba..31ebf2c 100644
--- a/device/serial/BUILD.gn
+++ b/device/serial/BUILD.gn
@@ -45,7 +45,7 @@ static_library("serial") {
     configs += [ "//build/config/linux:udev" ]
   }
 
-  deps = [
+  public_deps = [
     ":serial_mojo",
   ]
 }
