diff --git a/mojo/mojo_base.gyp b/mojo/mojo_base.gyp
index cb7a026..1d8505f 100644
--- a/mojo/mojo_base.gyp
+++ b/mojo/mojo_base.gyp
@@ -99,8 +99,8 @@
       'sources': [
         'environment/environment.cc',
         # TODO(vtl): This is kind of ugly. (See TODO in logging.h.)
-        "public/cpp/environment/logging.h",
-        "public/cpp/environment/lib/logging.cc",
+        "../third_party/mojo/public/cpp/environment/logging.h",
+        "../third_party/mojo/public/cpp/environment/lib/logging.cc",
       ],
       'include_dirs': [
         '..',
