diff --git a/mojo/mojo_nacl.gyp b/mojo/mojo_nacl.gyp
index 4007454..d79389e 100644
--- a/mojo/mojo_nacl.gyp
+++ b/mojo/mojo_nacl.gyp
@@ -49,6 +49,9 @@
           'sources': [
             '<(monacl_codegen_dir)/mojo_syscall.cc',
           ],
+          'dependencies': [
+            '../third_party/mojo/mojo_public.gyp:mojo_system_placeholder',
+          ],
         },
         {
           'target_name': 'monacl_syscall_win64',
@@ -62,6 +65,9 @@
           'sources': [
             '<(monacl_codegen_dir)/mojo_syscall.cc',
           ],
+          'dependencies': [
+            '../third_party/mojo/mojo_public.gyp:mojo_system_placeholder',
+          ],
           'configurations': {
             'Common_Base': {
               'msvs_target_platform': 'x64',
diff --git a/mojo/mojo_nacl_untrusted.gyp b/mojo/mojo_nacl_untrusted.gyp
index 4e7aab8..0cb4948 100644
--- a/mojo/mojo_nacl_untrusted.gyp
+++ b/mojo/mojo_nacl_untrusted.gyp
@@ -31,6 +31,7 @@
           ],
           'dependencies': [
             'mojo_nacl.gyp:monacl_codegen',
+            '../third_party/mojo/mojo_public.gyp:mojo_system_placeholder',
           ],
         },
         {
diff --git a/ppapi/ppapi_tests_mojo.gyp b/ppapi/ppapi_tests_mojo.gyp
index a7ec351..63a3e3a 100644
--- a/ppapi/ppapi_tests_mojo.gyp
+++ b/ppapi/ppapi_tests_mojo.gyp
@@ -41,6 +41,7 @@
       'dependencies': [
         '../mojo/mojo_nacl_untrusted.gyp:libmojo',
         '../mojo/mojo_nacl.gyp:monacl_codegen',
+        '../third_party/mojo/mojo_public.gyp:mojo_system_placeholder',
         '../native_client/src/untrusted/nacl/nacl.gyp:imc_syscalls_lib',
         'native_client/native_client.gyp:ppapi_lib',
         'ppapi_nacl.gyp:ppapi_cpp_lib',
