diff --git a/third_party/mojo_services/src/mojo_sdk_root.gni b/third_party/mojo_services/src/mojo_sdk_root.gni
index 1d31dcc..be420b7 100644
--- a/third_party/mojo_services/src/mojo_sdk_root.gni
+++ b/third_party/mojo_services/src/mojo_sdk_root.gni
@@ -4,3 +4,4 @@
 
 # This variable should point to the parent directory of the Mojo SDK.
 mojo_sdk_root = "//third_party/mojo/src"
+network_service_root = "//mojo/services"
