diff --git a/mojo/public/python/BUILD.gn b/mojo/public/python/BUILD.gn
index e2a2c33..f746cad 100644
--- a/mojo/public/python/BUILD.gn
+++ b/mojo/public/python/BUILD.gn
@@ -17,6 +17,7 @@ group("python") {
 python_binary_module("system") {
   python_base_module = "mojo"
 
+  public_configs = [ rebase_path("mojo/build/config", ".", mojo_root) + ":mojo_sdk" ]
   sources = [
     "mojo/c_core.pxd",
     "mojo/c_environment.pxd",
